doc/source/examples/03_modeling/scale_map_bodies.mystnb

```diff
@@ -0,0 +1,233 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: .venv
+  language: python
+  name: python3
+---
+
+# Design: Scale and Map Bodies
+
+The purpose of this notebook is to demonstrate the `map` and `scale` functions and their usage for transforming bodies.
+
+```{code-cell} ipython3
+# Imports
+import numpy as np
+from ansys.geometry.core import Modeler
+from ansys.geometry.core.math.point import Point2D, Point3D
+from ansys.geometry.core.sketch.sketch import Sketch
+from ansys.geometry.core.math.frame import Frame
+from ansys.geometry.core.math.vector import UnitVector3D
+from ansys.geometry.core.math.plane import Plane
+```
+
+```{code-cell} ipython3
+# Initialize the modeler for this example notebook
+m = Modeler()
+```
+
+## Scale Body
+
+The `scale` function is designed to modify the size of 3D bodies by a specified scale factor. This function is a important part of geometric transformations, allowing for the dynamic resizing of bodies.
+
+### Usage
+
+To use the `scale` function, you call it on an instance of a geometry body, passing a single argument: the scale value. This value is a real number (`Real`) that determines the factor by which the body's size will be changed.
+
+```python
+body.scale(value)
+
++++
+
+## Example: Making a Cube
+The code snippets below demonstrates how to change the size of a cube using the `scale` function in `body.py`. The process involves initializing a sketch design for the cube, defining the shape parameters, and then performing rescaling operation to generate the new shape.
```

Comment:
```suggestion
The following code snippets show how to change the size of a cube using the `scale` function in `body.py`. The process involves initializing a sketch design for the cube, defining the shape parameters, and then performing a rescaling operation to generate the new shape.
```
I'm not familiar with tagging (use of backticks) in MYSTNB files, so I'll have to get up to speed.